suite2p/run_s2p.py

```diff
@@ -177,22 +178,25 @@ def run_plane(ops, ops_path=None):
         print('----------- REGISTRATION')
         ops = registration.register_binary(ops) # register binary
         np.save(ops['ops_path'], ops)
-        print('----------- Total %0.2f sec'%(time.time()-t11))
+        plane_time_dict['registration'] = time.time()-t11
+        print('----------- Total %0.2f sec' % plane_time_dict['registration'])
```

Comment:
As I understand it, by including the printed time in the dict it makes it easier to analyze the timing data later on.  This is nice, but it would be great if it were applied consistently throughout the whole Suite2p package (wherever timing is measured and a print statement applied).  Alternatively, the print itself could be formatted more consistently, and then no dictionary would be necessary--the timing data could be parsed from the standard output.